mongo/check.py

```diff
@@ -624,6 +626,18 @@ def _parse_uri(self, server, sanitize_username=False):
 
         return username, password, db_name, nodelist, clean_server_name, auth_source
 
+    def _collect_indexes_stats(self, instance, db, tags):
```

Comment:
Would be nice if this function had a comment (perhaps indicating that this metric has to be fetched independently because it's not present in the "stats" command from mongo 😉 )